ui/boxpointer.js

```diff
@@ -0,0 +1,64 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
+/* exported enable, disable */
+/*
+ * Copyright © 2020 Endless OS Foundation LLC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { St } = imports.gi;
+
+const BoxPointer = imports.ui.boxpointer;
+
+const ExtensionUtils = imports.misc.extensionUtils;
+const PanelExtension = ExtensionUtils.getCurrentExtension();
+const Utils = PanelExtension.imports.utils;
+
+function enable() {
+    Utils.override(BoxPointer.BoxPointer, '_calculateArrowSide', function(arrowSide) {
+        const sourceTopLeft = this._sourceExtents.get_top_left();
+        const sourceBottomRight = this._sourceExtents.get_bottom_right();
+        const [boxWidth, boxHeight] = this.get_preferred_size();
```

Comment:
Did you mean to use minimal size here? Asking cause I'm not seeing any difference from the original impl [[ https://github.com/endlessm/gnome-shell/blob/master/js/ui/boxpointer.js#L589-L619 | here ]].

Reply:
Look closely :slightly_smiling_face: 

```diff
-let [, , boxWidth, boxHeight] = this.get_preferred_size();
+const [boxWidth, boxHeight] = this.get_preferred_size();
```

Reply:
Ahhh, totally missed it :).